website/content/en/docs/best-practices/pod-security-standards.md

```diff
@@ -393,7 +387,22 @@ var _ = Describe("my operator test", func() {
 })
 ```
 
- 
+### After following the recommendations to be restricted my workload is not running (CreateContainerConfigError). What should I do?
+
+If you are encountering errors similar to `Error: container has runAsNonRoot and image has non-numeric user` 
+or `container has runAsNonRoot and image will run as root` that means that the image used does not have a non-zero numeric user defined, i.e.:
+
+```shell
+USER 65532:65532 
+OR
+USER 1001
+```
+
+Due to the `RunAsNonRoot` field being set to `true`, we need to force the user in the
+container to a non-zero numeric user.
+It is recommended that the images used by your operator have a non-zero numeric user set in the image itself (similar to the example above).
+In vanilla Kubernetes you can also define the user via the security context configuration. (i.e. `RunAsUser: &[]int64{1000}[0],`). 
+It is important to note that defining the `RunAsUser` field will not work on K8s vendors like OpenShift, as the `RunAsUser` field violates the restricted-v2 SCC used by OpenShift. In this case the user will need to be defined in the image. 
```

Comment:
```suggestion
**NOTE** If your Operator will be distributed to other vendors please ensure that you check they have specific rules for Pod Security Admission.  For example, we know that if you use `RunAsUser` on OpenShift it will disqualify the Pod from their policy restricted-v2 SCC and values like 1001 would not work resulting in errors like `Leave the runAsUser field empty, or provide a value that falls within the specific user range for the namespace. provide a value in the ranges: [1000680000, 1000689999]`. Therefore, if you want your workloads running in namespaces labelled to enforce restricted you must leave empty this option and ensure that the image itself properly defines the UserID instead.  
```